pkg/types/kustomization.go

```diff
@@ -283,3 +286,13 @@ type KVSource struct {
 	Name       string     `json:"name,omitempty" yaml:"name,omitempty"`
 	Args       []string   `json:"args,omitempty" yaml:"args,omitempty"`
 }
+
+type Prune struct {
+	Type           string   `json:"type,omitempty" yaml:"type,omitempty"`
+	AlphaConfigMap NameArgs `json:"alphaConfigMap,omitempty" yaml:"alphaConfigMap,omitempty"`
```

Comment:
Commented on use of `Alpha` prefix above.  I think it looks odd, and it creates toil downstream because we'll have to change it.